code-challenges.test.js

```diff
@@ -24,11 +34,50 @@ const fibLength2 = 10
 
 // b) Create the function that makes the test pass.
 
+// Pseudocode:
+// 1. Create a function called greaterThanTwo
+// 2. Create a parameter called input. This input will be take in a number value
+// 3. Create an output array with index[0] and index[1] already defined. This output array is what we will use to return
+// 4. Create a for loop
+    // 4a. Create a new variable called 'i' that starts at 2 (& it starts at 2 because we have already defined index position 0 and 1 in the output array)
+    // 4b. Set the loop to stop once it has reached the input number
+    // 4c. Increment the variable i each iteration
+// 5. We are going to set the output index to 'i'
+    // 5a. Then we use math logic of adding the previous index and the second previous index together and save it the output index we are currently at
+// 6. Once the loop has reached the end, return the output array
+
+const greaterThanTwo = (input) => {
+    const output = [1, 1]
+    
+    for(let i=2; i<input; i++) {
+        output[i] = output[i-2] + output[i-1]
+        }
+    return output
+}
+
+console.log(greaterThanTwo(fibLength1)) // Output: [ 1, 1, 2, 3, 5, 8 ]
+console.log(greaterThanTwo(fibLength2)) // Output: [ 1,  1,  2,  3,  5, 8, 13, 21, 34, 55 ]
```

Comment:
No need for console logs when writing tests.  Logic looks great on this problem though!  Great work!